.circleci/config.yml

```diff
@@ -44,17 +44,17 @@ jobs:
       - run: bundle exec rake db:schema:load
 
       # run tests!
-      - run:
-          name: run tests
-          command: |
-            mkdir /tmp/test-results
-            TEST_FILES="$(circleci tests glob "spec/**/*_spec.rb" | circleci tests split --split-by=timings)"
+      #- run:
```

Comment:
Remove commented out code

Reply:
this part has to do with test and i havent started writing the test yet. When i do i will uncomment